aioitertools/builtins.py

```diff
@@ -159,7 +161,70 @@ def iter(itr: AnyIterable) -> AsyncIterator[T]:
     return value
 
 
-async def zip(*itrs: AnyIterable) -> AsyncIterator[Sequence[T]]:
+T1 = TypeVar("T1")
+T2 = TypeVar("T2")
+T3 = TypeVar("T3")
+T4 = TypeVar("T4")
+T5 = TypeVar("T5")
```

Comment:
Define these in `.types` and import them instead, for consistency with other TypeVars.